lib/std/http/Client.zig

```diff
@@ -160,9 +160,7 @@ pub const Connection = struct {
     proxied: bool = false,
     closing: bool = false,
 
-    read_start: u16 = 0,
-    read_end: u16 = 0,
-    read_buf: [buffer_size]u8 = undefined,
+    buffered: std.io.BufferedReaderWriter(std.crypto.tls.max_ciphertext_record_len, 4096, *Connection),
 
     pub fn rawReadAtLeast(conn: *Connection, buffer: []u8, len: usize) ReadError!usize {
```

Comment:
Since `readAtLeast` was removed this prefix can be dropped and this function can take it's place as `readAtLeast`.